bcda/models/models.go

```diff
@@ -251,7 +251,11 @@ func CreateACO(name string, cmsID *string) (uuid.UUID, error) {
 	db := database.GetGORMDbConnection()
 	defer database.Close(db)
 
-	aco := ACO{Name: name, CMSID: cmsID, UUID: uuid.NewRandom()}
+	id := uuid.NewRandom()
+
+	// TODO: remove ClientID below when a future refactor removes the need
```

Comment:
Has a ticket been created to address the `TODO`?

Reply:
> Has a ticket been created to address the `TODO`?

BCDA-1369 and BCDA-1370